conda-forge.yml

```diff
@@ -7,3 +7,8 @@ azure:
   settings_win:
     variables:
       CONDA_BLD_PATH: C:\bld\
+channels:
+  # channel to push to and the second element is the label on that channel
+  # adds ltr label for the LTR branch
+  targets:
+    - ["conda-forge", "ltr"]
```

Comment:
We need to add this to a `conda_build_config.yaml` file in the recipe directory:

```ytaml
channel_targets:
   - conda-forge ltr-qgis
```

This will allow users to install with `conda install -c conda-forge/label/ltr-qgis qgis`. Note that we need to add `qgis` at the end of the `ltr` label to prevent conda from getting all packages from an accidental "ltr" label and installing unexpected packages. For more info see https://github.com/conda-forge/cfep/blob/master/cfep-05.md

Also note that teh CFEP is for development versions, we never did this for an LTR before.

Reply:
@ocefpaf it must be both here and in the recipe? Or just in the recipe.

I don't want to do any ad hoc implementation. Should I open a proposal to the rest of the community? Not sure how many other packages have LTRs.

Reply:
> I don't want to do any ad hoc implementation. Should I open a proposal to the rest of the community? Not sure how many other packages have LTRs.

If you are OK with the label `ltr-qgis` instead of `ltr` we probably don't need a new proposal, we can just amend the current one. 

> @ocefpaf it must be both here and in the recipe? Or just in the recipe.

Sorry, I was not clear. There is no need to modify the recipe. All that is needed is to create a file named `conda_build_config.yaml` with that text inside the recipe directory. Then re-render and the label will be applied automatically.

Reply:
> If you are OK with the label `ltr-qgis` instead of `ltr` we probably don't need a new proposal, we can just amend the current one.

Thats fine by me. Just don't want do do anything that was not accepted by the community. I will try to propose a change in that CFEP.

> 
> > @ocefpaf it must be both here and in the recipe? Or just in the recipe.
> 
> Sorry, I was not clear. There is no need to modify the recipe. All that is needed is to create a file named `conda_build_config.yaml` with that text inside the recipe directory. Then re-render and the label will be applied automatically.

@ocefpaf I wasn't clear either. I meant in the recipe folder. We already have a conda_build_config.yaml file in the LTR branch so that I can pin some important dependencies. If I undestood correctly, I should revert the change I have made on conda-forge.yml  and implement it on conda_build_config.yaml, right?

Reply:
> If I undestood correctly, I should revert the change I have made on conda-forge.yml and implement it on conda_build_config.yaml, right?

Yep. But you already know that now :smile: